camp/mgetachew/mgetachewMiniProject/src/main/java/frc/robot/subsystems/MotorTalon.java

```diff
@@ -0,0 +1,91 @@
+package frc.robot.subsystems;
+
+import edu.wpi.first.wpilibj.motorcontrol.PWMTalonFX;
+import edu.wpi.first.wpilibj.motorcontrol.Talon;
+
+import com.ctre.phoenix6.configs.Slot0Configs;
+import com.ctre.phoenix6.configs.Slot1Configs;
+import com.ctre.phoenix6.StatusCode;
+import com.ctre.phoenix6.configs.TalonFXConfiguration;
+import com.ctre.phoenix6.controls.VelocityVoltage;
+import com.ctre.phoenix6.controls.VoltageOut;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import com.ctre.phoenix6.hardware.TalonFX;
+import frc.robot.Constants;
+import edu.wpi.first.wpilibj2.command.Command;
+import com.ctre.phoenix6.signals.InvertedValue;
+import com.ctre.phoenix6.signals.NeutralModeValue;
+
+public class MotorTalon extends SubsystemBase{
+    //configurtition for the motor begins here
+    private TalonFX motorTalon;
+    //velcity and voltage control
+    private final VelocityVoltage motorTalonVelocityController = new VelocityVoltage(0);
+    private final VoltageOut motorTalonVoltageController = new VoltageOut(0);
+    private TalonFXConfiguration motorTalonMotorConfiguration = null;
+    private Slot0Configs motorRpmGains = new Slot0Configs().withKP(0.36).withKI(0.1).withKD(0.0075).withKV(0.10);
```

Comment:
these are not the right gains to use for this project. That's why it's not driving smoothly.
What you need here is:
Slot0Configs().withKS(0.5);